pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_command-line-interface.adoc

```diff
@@ -0,0 +1,45 @@
+[id="command-line-interface_{context}"]
+
+= Using the Command Line Interface
+
+Many of the operations available in the Pantheon user interface can also be performed using commands in the command line interface.
+
+.To use the `Pantheon uploader` via the command line interface
+
+. Install the uploader script via setup.sh
++
+----
+curl -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh | sh -
+----
+. Type ‘pantheon’ in the command line to confirm it was properly installed.
+
+. Navigate to a directory containing test modules
++
+----
+cd <PATH_TO_YOUR_FOLDER_WITH_MODULES>
+----
+. Create the configuration file and edit as needed
++
+----
+# pantheon --sample > pantheon2.yml
+# vim pantheon2.yml  (or open in your preferred editor)
+----
+. Change the default values for ‘server’ and ‘repository’ as follows:
++
+server: link:http://pantheon2.int.us-west.aws.prod.paas.redhat.com[]
```

Comment:
Are we trying to avoid publishing internal Red Hat URLs?

Reply:
@benradey - yes.

@emarcusRH - can we change this to a more generic reference?